src/instructlab/sdg/generate_data.py

```diff
@@ -319,16 +331,18 @@ def generate_data(
     if generated_data is None:
         generated_data = []
 
-    _gen_train_data(logger, generated_data, os.path.join(output_dir, output_file_train))
+    _gen_train_data(
+        logger,
+        generated_data,
+        os.path.join(output_dir, output_file_generated_train),
+        os.path.join(output_dir, output_file_messages_train),
+    )
 
     # TODO
     # This is for backwards compatibility. The file existing previously, so we'll keep it for now.
     # I believe the github bot assumes it is present for presenting generated data to a taxonomy
     # reviewer or contributor. Otherwise, I don't see a consumer of it in this repo or the
     # `ilab` CLI.
-    _gen_train_data(
-        logger, generated_data, os.path.join(output_dir, output_file_generated)
-    )
```

Comment:
As discussed in #92 , this should remain

Reply:
changed the params so it is still saving the original file, it is called output_file_generated_train instead of output_file_messages_train.
@aakankshaduggal ran an e2e to confirm

Reply:
these 4 files are being generated, does that make sense @russellb 

![Screenshot 2024-07-08 at 3 04 07 PM](https://github.com/instructlab/sdg/assets/32435206/00d86ae4-1f74-4820-8940-35bc4885d299)

Reply:
FYI - this has changed the 3 files that will be saved are as follows:
`generated_Mixtral-8x7B-Instruct-v0_2024-07-08T18_23_10.json`
`test_Mixtral-8x7B-Instruct-v0_2024-07-08T18_23_10.jsonl`
`train_Mixtral-8x7B-Instruct-v0_2024-07-08T18_23_10.jsonl`